# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGSVGElement.cpp
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/svg/SVGSVGElement.cpp
--- a/dom/svg/SVGSVGElement.cpp	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/svg/SVGSVGElement.cpp	Thu Jun 20 07:03:54 2019 -0700
@@ -270,8 +270,8 @@
   return do_AddRef(new SVGMatrix());
 }
 
-already_AddRefed<SVGIRect> SVGSVGElement::CreateSVGRect() {
-  return NS_NewSVGRect(this);
+already_AddRefed<SVGRect> SVGSVGElement::CreateSVGRect() {
+  return do_AddRef(new SVGRect(this));
 }
 
 already_AddRefed<DOMSVGTransform> SVGSVGElement::CreateSVGTransform() {